app/src/main/java/ch/epfl/skysync/screens/LoadingScreen.kt

```diff
@@ -0,0 +1,24 @@
+package ch.epfl.skysync.screens
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import androidx.navigation.NavHostController
+import ch.epfl.skysync.components.LoadingComponent
+import ch.epfl.skysync.models.user.Admin
+import ch.epfl.skysync.navigation.Route
+import ch.epfl.skysync.viewmodel.FlightsViewModel
+
+@Composable
+fun LoadingScreen(navController: NavHostController, viewModel: FlightsViewModel) {
```

Comment:
I'm not sure to understand where this screen is used and what is its purpose ?

Reply:
It is for when you connect to the app it takes some time before we have the information if the user is an admin or not. So while it's not yet decided the loading screen appears, then when it's decided it takes you to the right home screen.

Reply:
I get it, we have to refactore `onSignInResult` anyway (I'll try to do it in #263), that's good in the mean time